CMakeLists.txt

```diff
@@ -24,6 +25,8 @@ include(CheckCCompilerFlag)
 include(CheckCXXCompilerFlag)
 include(CMakePushCheckState)
 
+# PEJ - I noted there are CFLAGS set here. These will trickle down to picotls
+#       unless moved to target compile options.  What's preferred?
 if(ENABLE_ASAN)
```

Comment:
Having the flags down to Picotls is probably fine. The ASAN and UBSAN flags are typically used in test builds, before running a set of tests, stress or fuzz cases to detect possible bad pointers, leaks, etc. Not a bad idea to also detect potential issues in Picotls.